docs/docs-content/clusters/pcg/deploy-app-pcg.md

```diff
@@ -0,0 +1,641 @@
+   | **NTP Servers**                                          | Skip.                                                                  | No                       |
+   | **SSH Public Keys**                                      | Provide a public OpenSSH key to be used to connect to the PCG cluster. | Yes                      |
+
+5. **PCG Cluster Size**
+
+   This tutorial will deploy a one-node PCG with dynamic IP placement (DDNS). If needed, you can convert a single-node
+   PCG to a multi-node PCG to provide additional capacity. Refer to the
+   [Increase PCG Node Count](./manage-pcg/scale-pcg-nodes.md) guide for more information.
+
+   | **Parameter**       | **Value**                                                                    | **Environment-Specific** |
+   | ------------------- | ---------------------------------------------------------------------------- | ------------------------ |
+   | **Number of Nodes** | `1`                                                                          | No                       |
+   | **Placement Type**  | `DDNS`                                                                       | No                       |
+   | **Search domains**  | Comma-separated list of DNS search domains. For example, `spectrocloud.dev`. | Yes                      |
+
+6. **Cluster Settings**
+
+   Set the parameter **Patch OS on boot** to `N`, meaning the OS of the PCG hosts will not be patched on the first boot.
+
+   | **Parameter**        | **Value** | **Environment-Specific** |
+   | -------------------- | --------- | ------------------------ |
+   | **Patch OS on boot** | `N`       | No                       |
+
+7. **vSphere Machine Configuration**
+
+   Set the size of the PCG as small (**S**) as this PCG will not be used in production environments.
+
+   | **Parameter** | **Value**                                     | **Environment-Specific** |
+   | ------------- | --------------------------------------------- | ------------------------ |
+   | **S**         | `4 CPU, 4 GB of Memory, and 60 GB of Storage` | No                       |
+
+8. **Node Affinity Configuration Information**
+
+   Set **Node Affinity** to `N`, indicating no affinity between Palette pods and control plane nodes.
+
+   | **Parameter**     | **Value** | **Environment-Specific** |
+   | ----------------- | --------- | ------------------------ |
+   | **Node Affinity** | `N`       | No                       |
+
+After answering the prompts of the `pcg install` command, a new PCG configuration file is generated, and its location is
+displayed on the console.
+
+```text hideClipboard
+==== PCG config saved ==== Location: /home/ubuntu/.palette/pcg/pcg-20240306182821/pcg.yaml
+```
+
+Next, Palette CLI will create a local kind cluster that will be used to bootstrap the PCG cluster deployment in your
```

Comment:
Can we put a link to https://kind.sigs.k8s.io/ to put the word kind into perspective?

Reply:
Yes!